beancount_import/source/ultipro_google_test.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pytest
+
+from . import ultipro_google
+from .source_test import check_source_example
+
+testdata_dir = os.path.realpath(
+    os.path.join(
+        os.path.dirname(__file__), '..', '..', 'testdata', 'source', 'ultipro_google'))
+
+examples = [
+    ('test_basic', '20210108-1.pdf'),
+]
+
+@pytest.mark.parametrize('name,ofx_filename', examples)
```

Comment:
Should be pdf_filename rather than ofx_filename

Reply:
This made me realize that the filename isn't used at all. Removed.